dogs-initial/src/main/java/com/lambdaschool/dogsinitial/controller/DogController.java

```diff
@@ -0,0 +1,87 @@
+package com.lambdaschool.dogsinitial.controller;
+
+import com.lambdaschool.dogsinitial.exception.ResNotFoundException;
+import com.lambdaschool.dogsinitial.model.Dog;
+import com.lambdaschool.dogsinitial.DogsinitialApplication;
+import com.lambdaschool.dogsinitial.model.MessageDetail;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.amqp.rabbit.core.RabbitTemplate;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+import org.springframework.web.servlet.ModelAndView;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Map;
+
+@RestController
+@RequestMapping("/dogs")
+public class DogController {
+
+    private static final Logger logger = LoggerFactory.getLogger(DogController.class);
+
+    @Autowired
+    RabbitTemplate rt;
+
+    // localhost:8080/dogs/dogs
+    @GetMapping(value = "/dogs")
+    public ResponseEntity<?> getAllDogs() {
+        logger.info("/dogs/dogs accessed");
+        MessageDetail message = new MessageDetail("/dogs/dogs accessed", 7, false);
+        rt.convertAndSend(DogsinitialApplication.QUEUE_NAME_HIGH, message);
+        return new ResponseEntity<>(DogsinitialApplication.ourDogList.dogList, HttpStatus.OK);
+    }
+
+    // localhost:8080/dogs/{id}
+    @GetMapping(value = "/{id}")
+    public ResponseEntity<?> getDogDetail(@PathVariable long id) {
+        logger.info("/dogs/" + id + " accessed");
+        MessageDetail message = new MessageDetail("/dogs/" + id + " accessed", 1, true);
+        rt.convertAndSend(DogsinitialApplication.QUEUE_NAME_LOW, message);
+        Dog rtnDog;
```

Comment:
You can add the date/time of access to your log/message text by concatenating `new Date()` to your log string (ex.: `new Date() + " endpoint accessed."`).